hug/authentication.py

```diff
@@ -197,13 +197,9 @@ def verify_jwt(authorization, response, jwt_secret):
     try:
         token = authorization.split(' ')[1]
         decoding = jwt.decode(token, jwt_secret, algorithm='HS256')
-        print(decoding)
         return decoding['user_id']
-    except Exception as ex:
-        template = "An exception of type {0} occurred. Arguments:\n{1!r}"
-        print(template.format(type(ex).__name__, ex.args))
+    except:
```

Comment:
Are you catching all exceptions ? 😮 Wouldn't this make things hard to debug ?  

Isn't there a specific exception that can be caught ?

Reply:
Yes it would, I'll go figure out which one/s I'm expecting to catch when the token is no longer valid. Thanks.

Reply:
@OGKevin  If an exception is thrown that isn't caught here what happens to the server? Does it continue running and print and error message, or does it break? Where is the code that handles the exceptions that get bubbled up?

Reply:
@dsmurrell The individual request will die, but the server itself will keep running, if you have an @hug.exception handler you can manage how that exception is handled